swagger/src/test/scala/org/http4s/rho/swagger/TypeBuilderSpec.scala

```diff
@@ -309,35 +323,86 @@ class TypeBuilderSpec extends Specification {
       }
     }
 
-    "Build an empty model for an AnyVal" in {
-      val ms = modelOf[FooVal]
-      ms must be empty
-    }
+    // fails because the implementation for AnyVals is just Seg.empty
+    "Treat AnyVals transparently" in {
+      modelOf[FooVal] must_== modelOf[Foo]
+    }.pendingUntilFixed("AnyVals")
```

Comment:
I agree its a problem that `AnyVals` are not treated transparently.  Do you intend to change this behavior?  I don't like the idea of just leaving these pending.  Do you mind filing an issue if you do not intend to change this as part of this PR?

Reply:
I didn't want to increase the scope of this PR, IMO it's a candidate for a separate one. I just didn't want it to be forgotten, so I wrote a test for it. I'll fill an issue, and if I'll find some time in next days, I'll submit a second PR.

Reply:
#272